test/e2e/case3_mutation_recovery_test.go

```diff
@@ -142,27 +142,6 @@ var _ = Describe("Test unexpected policy mutation", func() {
 			return plc.Object["spec"]
 		}, defaultTimeoutSeconds, 1).Should(utils.SemanticEqual(rootPlc.Object["spec"]))
 	})
-	It("Should recover root policy status if modified", func() {
-		By("Clearing the status of the root policy")
-		rootPlc := utils.GetWithTimeout(
-			clientHubDynamic, gvrPolicy, case3PolicyName, testNamespace, true, defaultTimeoutSeconds,
-		)
-		Expect(rootPlc).ToNot(BeNil())
-		rootPlc.Object["status"] = policiesv1.PolicyStatus{}
-		rootPlc, err := clientHubDynamic.Resource(gvrPolicy).Namespace(testNamespace).UpdateStatus(
-			context.TODO(), rootPlc, metav1.UpdateOptions{},
-		)
-		Expect(err).To(BeNil())
-		By("Getting root policy again")
-		yamlPlc := utils.ParseYaml("../resources/case3_mutation_recovery/managed-both-status-compliant.yaml")
-		Eventually(func() interface{} {
-			rootPlc = utils.GetWithTimeout(
-				clientHubDynamic, gvrPolicy, case3PolicyName, testNamespace, true, defaultTimeoutSeconds,
-			)
-
-			return rootPlc.Object["status"]
-		}, defaultTimeoutSeconds, 1).Should(utils.SemanticEqual(yamlPlc.Object["status"]))
-	})
```

Comment:
Is it possible to come up with a way that the root policy status would be re-calculated if it was deleted like this? I don't really like the idea of removing this behavior, but I do understand the trade-off

Reply:
I could change the predicate to reconcile when the status is empty but that only partially fixes the problem and could be wasteful.

What do you think?

Reply:
How wasteful would it be? Your suggestion seems like a quick and easy way to get this reconcile to happen. What's only partially fixed?